net-irc/znc/znc-9999.ebuild

```diff
@@ -2,163 +2,157 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-PYTHON_COMPAT=( python3_4 )
-inherit autotools eutils python-single-r1 user
+PYTHON_COMPAT=( python{3_4,3_5} )
+PLOCALES="ru"
+: ${CMAKE_MAKEFILE_GENERATOR:=ninja}
```

Comment:
I copied it from other ebuilds.
https://github.com/gentoo/gentoo/blob/ca375fd411897ae66f0691c9f8b257ed364e6e05/dev-util/creduce/creduce-2.6.0.ebuild#L7
https://github.com/gentoo/gentoo/blob/4a89b21f3910191a8b68e171092b05ebe1b571d3/sci-chemistry/votca-csgapps/votca-csgapps-1.4.ebuild#L7